src/OrchardCore.Themes/TheComingSoonTheme/Assets/css/bootstrap-oc.css

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
This Assets/css/ folder is not necessary. Seems like it generated files in there from a previous gulp build.